src/main/scala/com/gu/support/workers/lambdas/SendThankYouEmail.scala

```diff
@@ -50,6 +50,8 @@ class SendThankYouEmail(thankYouEmailService: EmailService, servicesProvider: Se
           edition = state.user.country.alpha2,
           name = state.user.firstName,
           billingPeriod = state.product.billingPeriod,
+          sfContactId = Some(state.salesForceContact.Id),
+          identityId = None,
```

Comment:
So if I understand correctly you only need *either* the sfContactId or the identity id but not both?
In which case it might be nice to use an `Either` rather than two options?

Reply:
I made the changes in f05ab53cd3c84756791b406aece73f8fa4049368